### ISSUE TITLE
Feature Request: Memory used/memory free statistic

### ISSUE THREAD

**Issue by ghalfacree** [mooltipass/moolticute#307]
I have a lot of passwords. I mean, a _lot_ of passwords. Now, I know I can store plenty of them in the 8Mb (1MB) Mooltipass Mini, but I'm not sure how many. In particular, I'm not sure how close I'm coming to the upper limit - which is stopping me from playing around with the file storage functionality in Moolticute, because I don't want to end up filling the thing.

Would it be possible to add a feature to Moolticute to show you how much of the Mooltipass' internal memory is used? I'd be happy with memory used (0.5Mb/8Mb used), memory free (7.5Mb free), or percentage used (6.25% used) or free (93.75% free). A progress bar, even. Just something so I have an idea how close I'm sailing to the wind.

Thanks!

**Comment by limpkin** [comment]
That could easily be implemented.
@netromdk if you feel like it, this could be done within the "integrity check" feature.

**Comment by netromdk** [comment]
Good idea, @ghalfacree.

@limpkin if it's part of the integrity check then some text would need to be added so convey that memory usage will be displayed. Perhaps this needs to be displayed "globally" in MC somewhere? like the title or the about tab?

**Comment by limpkin** [comment]
I don't think we can display it globally as we'll only be able to know the actual memory usage only when the memory integrity check is run.
It'd therefore be some kind of "added benefit" to that integrity check feature

**Comment by netromdk** [comment]
That makes sense. So it would be enough to add this information to the info dialog at the end of the integrity check?

**Comment by ghalfacree** [comment]
That'd work for me. Passwords are only small; it's the sort of thing I'd be checking once every couple of months, or just before I shove a large-ish file on there - not something I'd need to see daily.

**Comment by limpkin** [comment]
@netromdk yep.
glad we agreed on how to implement this :)

**Comment by netromdk** [comment]
Great! I will take a look at this soonish. :)
